testproxy/services/mutate-row.js

```diff
@@ -23,10 +23,11 @@ const mutateRow = ({clientMap}) =>
   normalizeCallback(async rawRequest => {
     const {request} = rawRequest;
     const {request: mutateRequest} = request;
-    const {appProfileId, mutations, tableName, rowKey} = mutateRequest;
```

Comment:
Eventually we will want the appProfileId to be able to be set per mutateRow call, but the change to use appProfileId to address the test is good for now.